lib/index.js

```diff
@@ -1,3 +1,6 @@
+// used to avoid exporting same module
```

Comment:
hoisting and `var` don't work like that and usually var declarations are all on top. Anyway, I don't mind this change.